org.freedesktop.Sdk.Extension.texlive.yaml

```diff
@@ -0,0 +1,106 @@
+  prefix: /usr/lib/sdk/texlive
+  env:
+    PATH: /usr/lib/sdk/texlive/bin:/usr/bin
+modules:
+
+- name: p7zip # required for unzipping texlive.iso
+  no-autogen: true
+  make-args:
+    - 7z
+  cleanup:
+    - '*'
+  sources:
+    - type: archive
+      url: https://sourceforge.net/projects/p7zip/files/p7zip/16.02/p7zip_16.02_src_all.tar.bz2
+      sha256: 5eb20ac0e2944f6cb9c2d51dd6c4518941c185347d4089ea89087ffdd6e2341f
+    - type: shell
+      commands:
+      - sed -i 's|/usr/local|/usr/lib/sdk/texlive|g' makefile.common
+
+- name: wget # build dependency of TeX Live
+  cleanup:
+    - '*'
+  sources:
+    - type: archive
+      url: https://ftp.gnu.org/gnu/wget/wget-1.19.tar.xz
+      sha256: 0f1157bbf4daae19f3e1ddb70c6ccb2067feb834a6aa23c9d9daa7f048606384
+
+- name: texlive
+  buildsystem: simple
+  sources:
+    - type: file
+      url: http://mirrors.ctan.org/systems/texlive/Images/texlive.iso
+      sha512: 7b7f0dd0eab3bfffe52c5cd1139c7f75d029b9ff4c4ce0e57e06834705522f4ec0c02cd99a80b053c6619abda51c20a60f8e91e91781bdc2b9b60fc2e5708adb
+  build-commands:
+    - 7z x texlive.iso
+    - chmod +x install-tl
+    - "echo \"I\nn\" | TEXLIVE_INSTALL_PREFIX=/usr/lib/sdk/texlive ./install-tl -portable"
+    - rm -rf /usr/lib/sdk/texlive/texmf-dist/doc
+
+- name: perl # required for makeglossaries
+  no-autogen: true
+  config-opts:
+  - -des # use default build configuration
+  - -Duseshrplib # build shared library
+  build-options:
+    cflags: -fPIC
+    ldflags: -fpic
+    no-debuginfo: true
+  cleanup:
+    - /man
+    - "*.pod"
+    - "*.h"
+  sources:
+  - type: archive
+    url: http://www.cpan.org/src/5.0/perl-5.26.2.tar.gz
+    sha256: 572f9cea625d6062f8a63b5cee9d3ee840800a001d2bb201a41b9a177ab7f70d
+  - type: script
+    dest-filename: configure
+    commands:
+    - exec ./configure.gnu $@
+
+- name: ghostscript # required for including .eps graphics
+  config-opts:
+  - --disable-cups
+  build-options:
+    ldflags: -L/app/lib
+  sources:
+  - type: archive
+    url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs923/ghostscript-9.23.tar.xz
+    sha256: 1fcedc27d4d6081105cdf35606cb3f809523423a6cf9e3c23cead3525d6ae8d9
+
+- name: appdata
+  buildsystem: simple
+  build-commands:
+    - mkdir -p ${FLATPAK_DEST}/share/appdata
+    - cp org.freedesktop.Sdk.Extension.texlive.appdata.xml ${FLATPAK_DEST}/share/appdata
+    - appstream-compose  --basename=org.freedesktop.Sdk.Extension.texlive --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.Sdk.Extension.texlive
+  sources:
+  - type: file
+    path: org.freedesktop.Sdk.Extension.texlive.appdata.xml
+
+- name: scripts
+  buildsystem: simple
+  build-commands:
+  - cp *.sh /usr/lib/sdk/texlive/
+  sources:
+  - type: script
+    commands:
+    - install -d /app
+    - cp -ra /usr/lib/sdk/texlive/* /app/
+    dest-filename: install.sh
+  - type: script
+    commands:
+    - export PATH=/usr/lib/sdk/texlive/bin/x86_64-linux:/usr/lib/sdk/texlive/bin:$PATH
```

Comment:
this script isn't marked x86-64 only. Is it going to work on arm and i386 as well?

Reply:
Probably not. I changed the flathub.json file.